ProductWarehouse.API/Program.cs

```diff
@@ -0,0 +1,53 @@
+using ProductWarehouse.Application.Extensions;
+using Serilog;
+using System.Reflection;
+using ProductWarehouse.Infrastructure.Extensions;
+using ProductWarehouse.API.Mapping;
+using ProductWarehouse.Persistence.Extensions;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddControllers();
+builder.Services.AddHttpClient();
+builder.Services.DependencyRegistration();
```

Comment:
```
builder.Services.DependencyRegistration();
builder.Services.DependencyRegistration(builder.Configuration);
```
I made a mistake
must have a suffix what we are registering like that is too abstract.
It's my fault